src/colmap/geometry/essential_matrix.cc

```diff
@@ -142,4 +142,10 @@ Eigen::Matrix3d InvertEssentialMatrix(const Eigen::Matrix3d& E) {
   return E.transpose();
 }
 
+Eigen::Matrix3d FundamentalFromEssentialMatrix(const Eigen::Matrix3d& K2,
```

Comment:
This function is correct. However, this reminds me that COLMAP only does undistortion at ``CamFromImg`` and ``ImgFromCam``. In such case, should we add a ``IsUndistorted`` check when applying the ``LORANSAC<FundamentalMatrixSevenPointEstimator, FundamentalMatrixEightPointEstimator>`` in ``EstimateCalibratedTwoViewGeometry`` here (https://github.com/colmap/colmap/blob/main/src/colmap/estimators/two_view_geometry.cc#L458).

Reply:
Yes, we don't estimate any distortion coefficients during two-view estimation but only do so in the later multi-view reconstruction process. If there are reliable solvers to do so, we could incorporate them into the two-view estimation procedure. Otherwise, I would not enforce any specific checks at the two-view stage, otherwise we prevent people from specifying known distortion parameters when reading images. Does that make sense?

Reply:
OK, I see what you mean now. I think that makes sense, but I would tackle it in a separate PR. Is this something you could add? Thanks.